### DISCUSSION TITLE
Is Websocket still not working?

### DISCUSSION THREAD

**Discussion by ClarissaGazalaEvanthe** [invertase/globe#32] [Q&A]
I want to build a chat app but I tried with Alfred and WebSocket couldn't just connect for a moment then disconnected,
When will websocket be supported?

**Comment by mrverdant13** [comment]
Why this discussion entry got closed without any explicit answer or reference? (authentic question, not criticism)

If this feature is not supported and there are no plans on doing so, it would be appreciated to explicitly be aware of that, imo.

**Reply by lesnitsky** [reply]
We do have plans on supporting WebSockets on Globe, a timeline is not very clear though. Feel free to follow [dart_globe](https://twitter.com/dart_globe) or hop on our [discord](https://discord.gg/UCcCYjDaTy) for updates.

**Reply by mrverdant13** [reply]
Sounds fair. Thanks for the reply.
